### ISSUE TITLE
Make the "planimetric measurements" option in CRS settings more explanatory

### ISSUE THREAD

**Issue by DelazJ** [qgis/QGIS#33528]
Under Settings --> Options --> CRS dialog, there's a "Planimetric measurements" checkbox implemented via #30952. This report is to mention some UI/UX behavior I think should be fixed to make the setting understandable (from what I could understand)
1. In the CRS settings dialog, choose a default crs to use for new projects
 check the planimetric measurement option
 reopen QGIS and the Options dialog
==>  the planimetric box is unchecked. Should it be?
1. from a user point of view, it's hard to guess what this feature does: there's no tooltip over a somehow "enigmatic" label, and if it applies to only the new projects with a default CRS set ( as I understand from the PR discussion), should it not be close to that setting?
Tested on ubuntu 18.04 wth QGIS 3.10.1 (1e6b1a3e)

**Comment by Pedro-Murteira** [comment]
@DelazJ I can no longer replicate this situation on QGIS 3.16.14 and 3.22.1.

**Comment by DelazJ** [comment]
Yes, the checkbox state is retained, but still remains the fact that what this checkbox does seems not obvious (unless it's just me)
